eNMS/server.py

```diff
@@ -123,6 +128,7 @@ def register_extensions(self):
 
     def configure_login_manager(self):
         login_manager = LoginManager()
+        login_manager.login_view = "blueprint.login"
```

Comment:
This is optional - it helps this feature work with an existing plugin that chose to use `@login_required` (flask_login) rather than `@server.monitor_requests`.  If this line is removed, we will just have to update that plug-in.

Reply:
the `monitor_requests` decorator is not optional, even for plugins. Not using it is a security issue. So this can be removed